vendor/tests/test_processor.py

```diff
@@ -296,6 +295,12 @@ class AuthorizeNetProcessorTests(TestCase):
 
     def setUp(self):
         self.existing_invoice = Invoice.objects.get(pk=1)
+        processor_site_config = SiteConfigModel()
```

Comment:
you could maybe set these values in the initial model declaration (e.g. `SiteConfigModel.objects.create(site=..., key=..., value=...)`

Reply:
I prefer to do it the other way jejejeje. Don't know if there is a preferred way by django standards